shared/config/constants.json

```diff
@@ -0,0 +1,31 @@
+{
+  "enums": [{
+    "comment": "Used to communicate different types of levels",
+    "contents": {
+      "LevelKind": {
+        "peer_review": "peer_review",
+        "assessment": "assessment",
+        "puzzle": "puzzle",
+        "unplugged": "unplugged",
+        "level": "level"
+      }
+    }
+  },
+
+  {
+    "comment": "Different possibilities for level.status, used to communicate how user has performed on a given level",
```

Comment:
Maybe that does make more sense. SharedConstants.rb is a static, human-modified file that is our source of truth. `generateSharedConstants.rb` then processes SharedConstants.rb and outputs `sharedConstants.js`